includes/class-wc-stripe-api.php

```diff
@@ -194,6 +194,16 @@ public static function retrieve( $api ) {
 	 * @return stdClass|array The response
 	 */
 	public static function request_with_level3_data( $request, $api, $level3_data, $order ) {
+		// Do not add level3 data for WC versions less than 3.0,
+		// since the store zip code is not available.
```

Comment:
Let's move this comment to `$level3_data = WC_Stripe_Helper::is_wc_lt( '3.0' ) ? :` calls.

Reply:
👌I have a comment in both places now.